internal/scorecard/alpha/scorecard.go

```diff
@@ -99,3 +122,30 @@ func ConfigDocLink() string {
 		"https://github.com/operator-framework/operator-sdk/blob/%s/doc/test-framework/scorecard.md",
 		version.Version)
 }
+
+// waitForTestsToComplete waits for a fixed amount of time while
+// checking for test pods to complete
+func waitForTestsToComplete(o Options, tests []ScorecardTest) (err error) {
+	for elapsedSeconds := 0; elapsedSeconds < o.WaitTime; elapsedSeconds++ {
+		allPodsCompleted := true
+		for i := 0; i < len(tests); i++ {
+			p := tests[i].TestPod
+			var tmp *v1.Pod
+			tmp, err = o.Client.CoreV1().Pods(p.Namespace).Get(p.Name, metav1.GetOptions{})
+			if err != nil {
+				fmt.Printf("error getting pod %s %s\n", p.Name, err.Error())
+				return err
+			}
+			if tmp.Status.Phase != v1.PodSucceeded {
+				allPodsCompleted = false
+			}
+
+		}
+		if allPodsCompleted {
+			return nil
+		}
+		time.Sleep(1 * time.Second)
+	}
+	return fmt.Errorf("error - wait time of %d seconds has been exceeded", o.WaitTime)
```

Comment:
I think the for inside for and the sleep is not required. See [here](https://github.com/operator-framework/operator-sdk/blob/master/pkg/test/e2eutil/wait_util.go#L54)

Reply:
I'd like to treat this as a TODO for the next PR if possible, but its worth considering.